qcengine/procedures/model.py

```diff
@@ -15,6 +15,9 @@ class Config:
         allow_mutation: False
         extra: "forbid"
 
+    def __init__(self, **kwargs):
+        super().__init__(**{**self._defaults, **kwargs})
```

Comment:
This causes an infinite recursion when I test this type of construct locally, and where is `_defaults` defined?